laborator/content/interactiune-c-assembly/README.md

```diff
@@ -152,8 +152,48 @@ Compilați și rulați programul. Pentru depanarea segfault-ului puteți folosi
 
 Adăugați în fișierul assembly instrucțiunile pentru conservarea și restaurarea registrelor necesare.
 
-### 8. Bonus: Calcul maxim în assembly cu apel din C pe 64 de biți
-Intrați în subdirectorul `8-max-c-calls-x64/` și faceți implementarea calculului maximului în limbaj de asamblare pe un sistem pe 64 de biți. Porniți de la programul de la exercițiile 4 și 5 în așa fel încât să îl rulați folosind un sistem pe 64 de biți.
+### 8. Warning (nu eroare)
+
+Accesați directorul `08-include-fix/`.
+Rulați comanda `make`, apare un warning, dar este de la procesul de preprocesare / compilare.
+Rezolvați acest warning prin editarea fișierului `hello.c`.
+
+Rezolvați warningul fără folosirea directivei `#include`.
+
+### 9. Reparare probleme de export
+
+Accesați directorul `09-export-fix/`.
+Fiecare subdirector (`a-func/`, `b-var/`, `c-var-2/`) conține o problemă legată de exportarea unor simboluri (funcții sau variabile).
+În fiecare subdirectorul, rulați comanda `make`, identificați problema și editați fișierele necesare pentru rezolvarea problemei.
+
+### 10. Tutorial: Linkare C și C++
+
+Accesăm directorul `10-cpp-obs/`.
+Vrem să urmărim cum se realizează linkarea din surse mixte: C și C++.
+
+În subdirectorul `bad/` avem două directoare `c-calls-cpp/` și `cpp-calls-c/` în care se combinăm surse mixte C și C++.
```

Comment:
I'd rename `bad/` and `good/` to `errors/` and `correct/` respectively. "Good" and "bad" are too vague.